### PR TITLE
Restructure Subscriber

### PR DESCRIPTION
**Pull request by jdtoombs**
Cleaning up the structure a little bit for implementation of new features 

- remove unused login components carried over from editor 
- move current subscriber code to more logical locations for future use

### CODE REVIEW

File: app/subscriber/src/components/footer/SubscriberFooter.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as styled from './styled';
 
-export const SubscriberFooter: React.FC<any> = () => {
```
Comment (Fosol) [inline app/subscriber/src/components/footer/SubscriberFooter.tsx:3]: Thanks for getting rid of `any`

**Review by Fosol — APPROVED** [review]
Next time you move files try and use the appropriate git move history options.  Sometimes it's not worth it because of changes in the files, but on a small change like this you could have done it.

`git add -A`
`git commit -m "Move files:"`

**Comment by Fosol** [comment]
@jdtoombs  You can try to regain history if you reapply your original commit with the instructions provided.

**Comment by jdtoombs** [comment]
@Fosol Think I got it - let me know :D

**Pull request merged**